static/docs/user-guide/index.md

```diff
@@ -27,6 +27,8 @@ guides related to contributing to this
   [DVC-file](/doc/user-guide/dvc-files-and-directories) dependencies.
 - [Managing External Data](/doc/user-guide/managing-external-data) explains both
   external outputs and external cache.
+- [Running DVC on Windows](/doc/user-guide/running-dvc-on-windows) contains useful information for our Windows users.
+- [Using Docker](/doc/user-guide/using-docker) explains how to set up a Docker image to develop with DVC and which considerations to have.
 - [Contributing](/doc/user-guide/contributing) is related with getting involved
   directly in developing DVC. Contributors are very welcomed in our
   [community](/support)!
```

Comment:
Thanks for adding `Running DVC on Windows` to this list @mroutis! 😄
- [ ] You're missing the sidebar.js entry for `Using Docker` though, so it shows up in the left side menu.
- [ ] Could you also move `Contributing` above, in the same order as the menu please?

Reply:
- [ ] Clicking on the `Using Docker` link, going to https://dvc-org-pr-811.herokuapp.com/doc/user-guide/using-docker, gives me a 404.

Reply:
Done, @jorgeorpinel , thanks for pointing those stuff!

Reply:
Just a quick sanity check :P